internal/app/api/server/middleware.go

```diff
@@ -205,54 +205,43 @@ func setSecureConfigEncryptor() gin.HandlerFunc {
 	}
 }
 
-func setServiceInfo() gin.HandlerFunc {
-	var mx sync.Mutex
-	mDB := make(map[uint64]*gorm.DB)
-	mSV := make(map[uint64]*models.Service)
-	getInstanceDB := func(c *gin.Context) (*gorm.DB, *models.Service) {
-		mx.Lock()
-		defer mx.Unlock()
-
-		var gDB *gorm.DB
-		if gDB = utils.GetGormDB(c, "gDB"); gDB == nil {
-			return nil, nil
-		}
+// Deprecated
+func setServiceInfo(db *gorm.DB) gin.HandlerFunc {
```

Comment:
This function seems to be used in two places in router (if I didn't miss anything), and looks like we can easily remove it right now, wdyt?

Reply:
Agree, I tried actually, but app crashed in random places, because `SV` key used in modules handlers, and modules handlers itself attached to several route groups, so it's better to refactor this part in separate PR